### PR TITLE
12.0 mig account invoice supplierinfo update

### PR DESCRIPTION
**Pull request by legalsylvain**
- [x] [MIG] port to version 12.0 (from 10.0)
- [x] [REF] hide button if supplier is not set. (partner_id is now optional in V12).
- [x] [FIX] the module was crashing in V8 and V10 if product was present in two lines.
- [x] [IMP] Store invoice currency on the supplierinfo ([Ref](https://github.com/OCA/account-invoicing/pull/412)) CC : @StefanRijnhart

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi all. just ported account_invoice_supplierinfo_update for 12.0. (trivial migration)

CC : @PierrickBrun, @mourad-ehm. 
@StefanRijnhart : I cherry-picked your commit present in this PR. https://github.com/OCA/account-invoicing/pull/412

thanks for your review.

**Review by PierrickBrun — APPROVED** [review]

File: account_invoice_supplierinfo_update/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Copyright (C) 2016-Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Invoice - Supplier Info Update',
+    'summary': 'In the supplier invoice, automatically updates all products '
+               'whose unit price on the line is different from '
+               'the supplier price',
+    'version': '10.0.1.0.1',
```
Comment (PierrickBrun) [inline account_invoice_supplierinfo_update/__manifest__.py]:
```suggestion
    'version': '12.0.1.0.0',
```

**Comment by legalsylvain** [comment]
thanks for your review @PierrickBrun. 
I applied your change.

**Pull request merged**